core/src/test/java/org/apache/iceberg/TestMetadataTableScans.java

```diff
@@ -356,14 +353,14 @@ public void testPartitionsTableScanNoFilter() {
                     Types.StructType.of(optional(1000, "data_bucket", Types.IntegerType.get()))))
             .asStruct();
     TableScan scanNoFilter = partitionsTable.newScan().select("partition.data_bucket");
-    Assert.assertEquals(expected, scanNoFilter.schema().asStruct());
+    assertThat(scanNoFilter.schema().asStruct()).isEqualTo(expected);
 
     CloseableIterable<ManifestEntry<?>> entries =
         PartitionsTable.planEntries((StaticTableScan) scanNoFilter);
     if (formatVersion == 2) {
-      Assert.assertEquals(8, Iterators.size(entries.iterator()));
+      assertThat(entries.iterator()).toIterable().hasSize(8);
```

Comment:
can't we just use `assertThat(entries).hasSize(xx)` here?

Reply:
This and other parts can be `assertThat(entries).hasSize(x)`. Will replace them.